### PR TITLE
add service defaults to Host

### PR DESCRIPTION
**Pull request by rysweet**
add aspire + Otel service defaults

### CODE REVIEW

File: dotnet/src/Microsoft.AutoGen.Agents.Runtime/Host.cs [hunk 1]
```diff
@@ -7,6 +8,7 @@ public static class Host
     public static async Task<WebApplication> StartAsync(bool local = false)
     {
         var builder = WebApplication.CreateBuilder();
+        builder.AddServiceDefaults();
```
Comment (LittleLittleCloud) [inline dotnet/src/Microsoft.AutoGen.Agents.Runtime/Host.cs:11]: Are we always expecting the runtime host to run in a seperete process?

Comment (rysweet) [reply dotnet/src/Microsoft.AutoGen.Agents.Runtime/Host.cs:11]: no, its explicitly in the same process (different thread and WebApplication) if local = true here.

**Comment by rysweet** [comment]
Note that you aren't required to use the Host and App - these are just conveniences for the DI for starting the services. We will do another sample that shows how to start them manually.

**Comment by rysweet** [comment]
closing - already merged from another branch

**Pull request closed without merging**